includes/class-freemius.php

```diff
@@ -20992,22 +20992,38 @@ private function _sync_plugin_license(
                              */
                             $api = $this->get_api_site_scope();
 
-                            if ( ! self::$_global_admin_notices->has_sticky( 'api_blocked' ) ) {
-                                self::$_global_admin_notices->add(
-                                    sprintf(
-                                        $this->get_text_inline( 'Your server is blocking the access to Freemius\' API, which is crucial for %1$s synchronization. Please contact your host to whitelist %2$s', 'server-blocking-access' ),
-                                        $this->get_plugin_name(),
-                                        '<b>' . implode( ', ', $this->apply_filters( 'api_domains', array(
-                                            'api.freemius.com',
-                                            'wp.freemius.com'
-                                        ) ) ) . '</b>'
-                                    ) . '<br> ' . $this->get_text_inline( 'Error received from the server:', 'server-error-message' ) . var_export( $result->error, true ),
-                                    $this->get_text_x_inline( 'Oops', 'exclamation', 'oops' ) . '...',
-                                    'error',
-                                    $background,
-                                    'api_blocked'
-                                );
-                            }
+	                        $add_notice = false;

```

Comment:
@msladu the entire code from this line and below doesn't seem to be properly indented.

Reply:
@vovafeldman It looks proper in my IDE setup.

Reply:
@vovafeldman Checking it.